pyproject.toml

```diff
@@ -88,6 +88,8 @@ filterwarnings = [
     # https://github.com/lanpa/tensorboardX/issues/653
     # https://github.com/lanpa/tensorboardX/pull/654
     "ignore:Call to deprecated create function:DeprecationWarning:tensorboardX",
+    # https://github.com/kornia/kornia/issues/777
+    "ignore:Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0:UserWarning:torch.nn.functional",
```

Comment:
This is only needed because our tests are trying to crop a 2x2 patch from a 1x1 image, which obviously requires resizing. If someone ever implements a `data.py` for GID-15 and increases the image size, this can be removed.